jsonschema/tests/test_jsonschema_test_suite.py

```diff
@@ -87,6 +87,24 @@ def narrow_unicode_build(test):  # pragma: no cover
         return
 
 
+if sys.version_info < (3, 7):
+    message = "Not running tests which require Python 3.7 or greater"
```

Comment:
Would rather keep this specific to the specific-reason we've got at the minute, i.e.:

"datetime.date.fromisoformat is new in 3.7" as the message, and as the name of the skipper something like `missing_date_fromisoformat`.

If there's other things that turn out missing in 3.6 we'll just add other skippers.